src/assets/stylesheets/variables/_layout.scss

```diff
@@ -0,0 +1,6 @@
+$grid-columns: 12;
+$grid-gutter-width: 30px;
+
+// use higher zindex, to avoid edit marker to be shown
+$zindex-modal-backdrop: 100040 !default;
+$zindex-modal: 100050 !default;
```

Comment:
I looked hard, but if I'm not mistaken, the `$zindex-modal-backdrop` and `$zindex-modal` are only used for the `modal` css class by Bootstrap. If I'm not mistaken this css class is never used.

So I guess we can actually get rid of all variables in `_layout.scss`. What do you think?

Reply:
Sure, I will just remove it then.